README.md

```diff
@@ -6,9 +6,13 @@ BaSta-LedControl is an Arduino program used to control the red and yellow basket
 
 BaSta-LedControl is intended to be used on an Arduino Uno board. The Stramatel basketball protocol is received through serial input. Two digital output pins control the LED state e.g. via switch relays.
 
+The Rx+ and the Rx- Pin of the Stramatel Remote is connected with a MAX485 TTL to RS485 Switch.
```

Comment:
I would prefer to not specifically mention the Stramatel Remote since the origin of the Stramatel Protocol may be another source entirely. E.g. we're using GameControl/ScoreboardRegie2009 software from LED Visual Sport Services that output the protocol via a local COM port on a Windows machine - usually with an FTDI RS485 usb adapter.